.github/workflows/python-package.yml

```diff
@@ -185,6 +185,9 @@ jobs:
     needs: isolated_tests
     if: startsWith(github.ref, 'refs/tags/')
     runs-on: ubuntu-latest
+    environment:
+      name: deploy
```

Comment:
Technically, the deployment target is PyPI, not Deploy. And that would differ from TestPyPI. So I tend to suggest those names that correspond to things they are supposed to represent.

Reply:
I had also thought of `PyPI`, but I didn't know if anything was standard and wanted to keep things simple since I didn't know that the environment would be created automatically anyways.

Reply:
You can reconfigure whenever. I use `pypi` and `testpypi` everywhere. Just remember to also update the trust config in PyPI.